fdm-app/app/routes/signin._index.tsx

```diff
@@ -165,332 +218,1143 @@ export default function SignIn() {
+                                                <p className="mt-1 text-sm text-muted-foreground">
+                                                    Als open-source platform
+                                                    faciliteren we kennisdeling
+                                                    en versnellen we innovatie.
+                                                </p>
+                                            </div>
+                                        </div>
+                                        <div className="flex items-start gap-4">
+                                            <div className="mt-1 flex h-8 w-8 shrink-0 items-center justify-center rounded-full bg-green-100 text-green-600">
+                                                <CheckCircle2 className="h-5 w-5" />
+                                            </div>
+                                            <div>
+                                                <h4 className="font-semibold">
+                                                    In ontwikkeling
+                                                </h4>
+                                                <p className="mt-1 text-sm text-muted-foreground">
+                                                    Een groeiend platform dat
+                                                    continu wordt verbeterd op
+                                                    basis van uw feedback.
+                                                </p>
+                                            </div>
+                                        </div>
+                                    </CardContent>
+                                </Card>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div className="bg-background py-24">
+                    <div className="container mx-auto max-w-6xl px-4 lg:px-8">
+                        <div className="mx-auto mb-20 max-w-3xl text-center">
+                            <h2 className="mb-6 text-3xl font-bold tracking-tight text-foreground sm:text-4xl">
+                                Doelsturing: Kansen en Uitdagingen
+                            </h2>
+                            <p className="text-lg leading-relaxed text-muted-foreground">
+                                Doelsturing kan een effectieve methode zijn om
+                                de waterkwaliteit te verbeteren en
+                                ammoniakemissies te verminderen, doordat u als
+                                ondernemer zelf de maatregelen kiest. De
+                                keerzijde is dat dit vraagt om veel en
+                                gedetailleerde data. {clientConfig.name} streeft
+                                ernaar om deze complexe berekening zo eenvoudig
+                                mogelijk te maken, zonder in te leveren op
```

Comment:
streeft ernaar om helder en concreet voor landbouwers in beeld te brengen wat de opgaven zijn en hoe zij kunnen sturen op een duurzame en toekomstbestendig landbouwsysteem? (als vervanging van "deze complexe berekening"?

Reply:
Ok, maar het is dan onduidelijk waarom je deze tool zou gebruiken en waarin het je helpt. Immers de website van de overheid vertelt je ook wat de opgave is, maar deze tool maakrt het ook echt concreet voor je bedrijf op een zo toeganglijke mogelijke manier